sdk/ai/azopenaiextensions/client_shared_test.go

```diff
@@ -191,19 +213,102 @@ type stainlessTestClientOptions struct {
 	UseAPIKey bool
 }
 
+func getRecordingOptions(t *testing.T) *recording.RecordingOptions {
```

Comment:
Is this required? It's a duplicate of normal test proxy code, correct? Do we need to export something more/different from the shared test recording package?